seed-db.ts

```diff
@@ -1246,13 +1255,14 @@ void (async function setupDevDB() {
             stepName: '',
             type: achievement_type_enum.STREAK,
             subtitle: 'Nachhilfe mit {{matchpartner}}',
-            description: 'Dieser Text muss noch geliefert werden.',
-            image: 'gamification/achievements/tmp/finished_course_sucessfully/finished_course_sucessfully.jpg',
-            achievedImage: 'Hat_gold',
-            actionName: null,
+            description:
+                'Du hast {{num}} Woche(n) in Folge mit {{name}} gelernt! Um diese Serie aufrechtzuerhalten, setze deine gemeinsamen Lernsessions mit {{name}} weiter fort. Regelmäßiges Lernen bringt eine Fülle an Vorteilen mit sich, von verbessertem Wissen und Verständnis bis hin zu gesteigerter Effizienz und Selbstvertrauen. Ihr seid definitiv auf dem richtigen Weg, um eure Ziele zu erreichen!',
```

Comment:
`{{ num }}` doesnt exist as variable, right?

Reply:
changed it to progress